civihr_employee_portal/tdd/HelperFunctionsTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require_once __DIR__ . '/../views/civihr_employee_portal.views.inc';
+
+class HelperFunctionsTest extends PHPUnit_Framework_TestCase
```

Comment:
One of the problems we have is the lack of tests for our Drupal code, so +1 for adding this. I can see some problems with it, but since we don't know exactly what are the "best practices" in terms of Drupal testing, I think it's fine to keep this test. 

Just for the record, these are the problems I see:
- You're testing a helper function. If this was a class, this function would probably be private and we would not need a specific test for it. This might be a hint that we don't need a test for `get_where_parts`
- You're using a require_once to include the views file. This is not such a huge problem, but the test will break if we rename/move the views file or the test
- Having a dir named `tdd` is just an example that we have no idea of what are the "best practices" for tests in Drupal.  This dir will certainly have to be renamed (and then, we might have problems with the require_once)
- The test doesn't cover the case where the field is an instance of `DatabaseCondition` (again, fine, because we don't know the best way to mock this)